CHANGELOG/v1.7.4.md

```diff
@@ -0,0 +1,51 @@
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.26.x -> v1.30.x
+- Workload Cluster: v1.24.x -> v1.30.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Changes since v1.7.3
+## :chart_with_upwards_trend: Overview
+- 22 new commits merged
+- 5 feature additions ✨
+- 5 bugs fixed 🐛
+
+## :sparkles: New Features
+- CABPK: Add kubeadm v1beta4 types (#10709)
```

Comment:
```suggestion
- CABPK: Add kubeadm v1beta4 types (#10771)
```

(that's my fault, should have dropped the link from the commit: https://github.com/kubernetes-sigs/cluster-api/pull/10771/commits)

Reply:
Done